src/test/java/com/bakdata/util/seq2/PairSeqTest.java

```diff
@@ -138,6 +182,26 @@ void shouldFlatMapValuesToPair() {
                 .containsExactlyInAnyOrder(new Tuple2<>(3, "a"), new Tuple2<>(4, "b"), new Tuple2<>(5, "c"));
     }
 
+    @Test
+    void shouldFlatMapValuesToOptionalPair() {
```

Comment:
Does the name make sense? If I understand the test correctly, it `flatMapOptionalValuesToPair`. Correct?
This sounds to me like the resulting pair is a Tuple of Optional.

Reply:
It maps the value of the PairSeq to an optional pair: the function returns `Optional<Tuple2<Integer, String>>`

Reply:
So yes it is correct